src/routes/profile-route/profile-route.jsx

```diff
@@ -9,20 +11,22 @@ export const ProfileRoute = () => {
   });
 
   return (
-    <div className="max-w-[1440px] min-h-[1023px] bg-gray-background mx-auto">
+    <div className="max-w-[1440px] bg-gray-background mx-auto min-h-screen relative">
```

Comment:
The header is currently changing to be larger than the max width of 1440px. I'd suggest removing it from this for now.